src/translations/nb.ts

```diff
@@ -485,7 +485,11 @@ const translations = {
   'grunnlag.title': 'Øvrig grunnlag for beregningen',
   'beregning.avansert.avbryt_modal.title':
     'Hvis du går ut av Avansert, mister du alle valgene dine.',
+  'beregning.avansert.avbryt_modal.endring.title':
```

Comment:
@Lucas8448 Tenkte å tagge deg her for å gjøre deg oppmerksom på at det kommer regelmessig nye tekster i nb, nn, og en filer. Jeg vil tro at Github skal håndtere det på en grei måte når du merger main i sandbox, men det er lurt å følge tett på slik at vi ikke mister tekster underveis :)